sphinx/builders/html/__init__.py

```diff
@@ -468,8 +468,8 @@ def prepare_writing(self, docnames: Set[str]) -> None:
         else:
             self.last_updated = None
 
-        logo = path.basename(self.config.html_logo) if self.config.html_logo else ''
-        favicon = path.basename(self.config.html_favicon) if self.config.html_favicon else ''
+        logo = self.config.html_logo if self.config.html_logo else ''
+        favicon = self.config.html_favicon if self.config.html_favicon else ''
```

Comment:
Could you add testcase for `html_logo` and `html_favicon` please?

Reply:
Ah yes, I'll look into how to best do that.